wiki/World.md

```diff
@@ -0,0 +1,140 @@
+# World #
+
+## General ##
+[World](https://github.com/RoboTeamTwente/roboteam_ai/blob/development/include/roboteam_ai/world_new/World.hpp#L36) by itself is a very interesting conversation, because we don't just have a single world.
+We have two worlds, one called [roboteam_world](http://github.com/roboteamtwente/roboteam_world), and one that we refer 
+to as [ai world](https://github.com/RoboTeamTwente/roboteam_ai/blob/development/include/roboteam_ai/world_new/World.hpp)
```

Comment:
Maybe unrelated, but I couldn't help when reading this to feel that one of these two things needs a rename to avoid confusion. Even during the discussion today Jesse was confused. So I would make this part of our advice.

Reply:
very true, we should. Roboteam Data maybe?

Reply:
Roboteam (pre)process?